### ISSUE TITLE
Is this project a currently executable one? Trapped by "Table 'chunyu_doctor.raw_html_illness' doesn't exist"

### ISSUE THREAD

**Issue by DongdongY1** [TTTTsao/chunyu_doctor#1]
I'm currently working on chunyu-doc medical text analysis and very surprised that here exists such a repo. Since no doc is provided and there is always an error "Table 'chunyu_doctor.raw_html_illness' doesn't exist"(after I tried to run "spider/db/create_all.py"), I am wondering about the state of this project. Please inform me of it and I would be very grateful ;)

**Comment by TTTTsao** [comment]
Add "illness_info" to "spider/db/tables.py __ all __ "

**Comment by DongdongY1** [comment]
> 

However it seems that "main.py" still could not run properly. I notice there are many tasks in crawl_task.py. Should they be executed in specified order? Or...

**Comment by TTTTsao** [comment]
Yes. You should crawl doctor_id first, then crawl question_id by doctor_id, crawl question page by question_id. You can send me your email address, will e-mail you a question_id sql.

**Comment by DongdongY1** [comment]
> 

It's okay to just paste it here :)
